audit.py

```diff
@@ -133,7 +133,10 @@ def process_json(repo_name, js):
     for kind in js["warnings"].values():
         for warn in kind:
             adv = warn["advisory"]
-            tup = repo_name, adv["package"], adv["id"]
+            if adv:  # Because this may be `None`.
```

Comment:
I think it would be clearer to say `if adv is not None:` than have the comment?

Reply:
Try 01d6b5b